test/multiverse/suites/ruby_openai/ruby_openai_instrumentation_test.rb

```diff
@@ -68,8 +68,30 @@ def test_chat_completion_records_summary_event
     summary_events = events.filter { |event| event[0]['type'] == NewRelic::Agent::Llm::ChatCompletionSummary::EVENT_NAME }
 
     assert_equal 1, summary_events.length
+  end
 
-    # TODO: Write tests that validate the event has the correct attributes
+  def test_chat_completion_events_assign_all_attributes
+    in_transaction do
+      stub_post_request do
+        client.chat(parameters: chat_params)
+      end
+    end
+    _, events = @aggregator.harvest!
+    summary_events = events.filter { |event| event[0]['type'] == NewRelic::Agent::Llm::ChatCompletionSummary::EVENT_NAME }
+    attributes = summary_events[0][1]
+
+    assert attributes['id']
+    assert attributes['span_id']
+    assert attributes['trace_id']
+    assert attributes['response.model']
+    assert attributes['vendor']
+    assert attributes['ingest_source']
+    assert attributes['request_max_tokens']
+    assert attributes['response.number_of_messages']
+    assert attributes['request.model']
+    assert attributes['response.choices.finish_reason']
+    assert attributes['request.temperature']
+    assert attributes['duration']
```

Comment:
Does `assert` ensure that each of the values would survive an `#empty?` check or simply a `#nil?` one?

Reply:
Great callout. I'll change these `assert`s to `refute_empty` for the hash values.

Reply:
@fallwith changed this https://github.com/newrelic/newrelic-ruby-agent/pull/2535/commits/46c2dfb008c5f02de116af9b205926ac5bdd0303